Teapot/Teapot.swift

```diff
@@ -274,7 +274,7 @@ open class Teapot {
 
         if let headerFields = headerFields {
             for headerField in headerFields {
-                if headerField.key == "Content-Type" {
+                if headerField.key.lowercased() == "content-type" {
```

Comment:
why lowercased ?

Reply:
Is that in case someone uses weird capitalization for `CoNTenT-tYPe` or some shit?

Reply:
Ah, also there's an [open swift bug](https://bugs.swift.org/browse/SR-2429) on this being case-sensitive